flow/payload.yaml

```diff
@@ -0,0 +1,48 @@
+components:
+  schemas:
+    Flow.Payload:
+      description: |-
+        A container for different types of payload, which is the data in the frame after protocol headers.
+        Some part of the payload will be overwritten with instrumentation data, contents and placement of 
```

Comment:
"will be" --> "could be" ?  [ e.g. metrics not enabled so no instrumentation data inserted or other implementations might have other ways to correlate Tx/Rx information ]